### PR TITLE
Remove unused components and update styles

### PR DESCRIPTION
**Pull request by mrruby**

### CODE REVIEW

File: src/main/index.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ import {
 } from '../types';
 import { checkHolochainLairBinariesExist } from './binaries';
 import { validateArgs } from './cli';
+import { searchHeigh, windowSize } from './const';
```
Comment (matthme) [inline src/main/index.ts]: This is a matter of taste but I like global constants to be all capital (e.g. `SEARCH_HEIGHT`) to make it clear that they are not dynamically assigned somewhere based on some logic but hard-coded.

**Review by matthme — APPROVED** [review]
There is currently no way to toggle light/dark mode explicitly, right? Becuase on Linux (and I think also Windows?) there is no OS level light/dark mode setting I think so that might be useful.

File: src/main/index.ts (hunk 1 above)
Comment (mrruby) [reply src/main/index.ts]: Let's keep all capital convention, I will change that

**Comment by mrruby** [comment]
I will ask Aaron to put the toggle into designs

**Pull request merged**